app/queries/best_sellers_service.rb

```diff
@@ -0,0 +1,9 @@
+class BestSellersService
+  def initialize(limit = Book::BESTSELLERS_LIMIT)
+    @limit = limit
+  end
+
+  def call
+    Book.left_joins(:orders).group(:id).order(Arel.sql('COUNT(orders.id) DESC')).limit(@limit)
```

Comment:
it seems this query doesn't work correctly, please add proper tests for this query
you should take into account quantity of books which was bought
check bookstore requirements